eng/versioning/version_client.txt

```diff
@@ -4,7 +4,7 @@
 com.azure:azure-sdk-all;1.0.0;1.0.0
 com.azure:azure-sdk-parent;1.6.0;1.6.0
 com.azure:azure-client-sdk-parent;1.7.0;1.7.0
-com.azure:azure-ai-anomalydetector;1.0.0-beta.1;1.0.0-beta.1
+com.azure:azure-ai-anomalydetector;1.0.0-beta.1;3.0.0-beta.1
```

Comment:
current and dependency both need to be 3.0.0-beta.1 for first release of package.